static/app/views/starfish/views/spans/spanSummaryPanel/useSpanSeenEvent.tsx

```diff
@@ -0,0 +1,67 @@
+import {useQuery} from 'sentry/utils/queryClient';
+import {useQueryGetEvent} from 'sentry/views/starfish/modules/databaseModule/queries';
+import {HOST} from 'sentry/views/starfish/utils/constants';
+import type {Span} from 'sentry/views/starfish/views/spans/spanSummaryPanel/types';
+
+type HasTransaction = {
+  transaction_id: string;
+};
+
+function useFirstSeenSpan(span?: Span, referrer = 'first-seen-span') {
```

Comment:
Small nits, but for consisntency we've been prefixing query specific hooks with `useQuery`
We haven't actually standardized this so up to you to resolve

Also, I like the idea of passing in what the function needs vs what you expect to pass into it. Seems a bit more generic, say you only have the group_id.
```suggestion
function useQueryFirstSeenSpan(span?: Pick<Span, 'group_id'>;, referrer = 'first-seen-span') {
```
Would also need to update the other one below.